src/wallet/wallet.cpp

```diff
@@ -3297,6 +3297,47 @@ bool CWallet::CreateZerocoinSpendModel(string &stringError, string thirdPartyAdd
 
 }
 
+bool CWallet::CreateZerocoinSpendModel(CWalletTx& wtx, string &stringError, string thirdPartyAddress, vector<string> denomAmounts, bool forceUsed) {
```

Comment:
I think it should be `const string& thirdPartyAddress` and `const vector<string>& denomAmounts` instead to prevent create another object.